x-pack/plugins/stack_connectors/public/connector_types/cases_webhook/webhook_connectors.test.tsx

```diff
@@ -96,6 +98,27 @@ describe('CasesWebhookActionConnectorFields renders', () => {
     expect(await screen.findByTestId('webhookCreateCommentJson')).toBeInTheDocument();
   });
 
+  it('Add comment to case section is rendered only when the toggle button is on', async () => {
```

Comment:
Maybe add a test that guarantees that `webhookAddCommentToggle` is active when those fields have data.

Reply:
done, thanks :)